api/src/main/java/org/openmrs/module/commonreports/CommonReportsConstants.java

```diff
@@ -27,6 +27,10 @@
 	
 	public static final String COMPONENT_REPORTMANAGER_OPDCONSULT = MODULE_ARTIFACT_ID + ".outpatientConsultation";
 	
+	public static final String COMPONENT_REPORTMANAGER_ANTENATALGESTATION = MODULE_ARTIFACT_ID + ".antenatalGestation";
+	
+	public static final String COMPONENT_REPORTMANAGER_ANTENATALRISKS = MODULE_ARTIFACT_ID + ".antenatalRisks";
+	
```

Comment:
You can actually use one report (Antenatal) and two datasets (Gestation & Risks), such that they don't appear to be separate from each other.

Reply:
Done